set_tags/utils.py

```diff
@@ -251,25 +251,55 @@ def get_marketplace_customer_id(synapse_id):
 
   return customer_id
 
+def get_marketplace_product_code(synapse_id):
```

Comment:
Why not combine this with get_marketplace_customer_id() so you get the customer Id and product code in a single database call?

Reply:
done